frontend/src/sass/layout/_nav.scss

```diff
@@ -2,7 +2,30 @@
    LAYOUT / NAV
    ================================= */
 
+.main-menu {
+    flex: auto;
```

Comment:
same than with `header` don't we have `nav` as parent?... maybe it exists on html, but not needed to style?